packages/recommend-js/src/utils/withHtml.tsx

```diff
@@ -11,7 +11,7 @@ export function withHtml<TProps>(
     <Component {...props} html={html} />
   );
 
-  if (__DEV__) {
+  if (process.env.__DEV__) {
```

Comment:
This won't get picked up by Babel and Rollup. I thought we had the tooling to support `__DEV__` in this repo already.

You need these additions:

- https://github.com/algolia/predict/blob/4537902a610cbc972a5e1a39a885997a0bc38de0/packages/predict-core/global.d.ts
- https://github.com/algolia/predict/blob/4537902a610cbc972a5e1a39a885997a0bc38de0/babel.config.js#L36-L45

But let's do that in another PR then.

Reply:
@SophieManley03 I'd recommend doing this in a separate PR now, merge it to `next`, and rebase this branch.

Reply:
Sure I can do it now. 
The purpose of it is to make the production bundle smaller?

Reply:
Yes, this way users don't pay the bundle price on their production apps.

Reply:
Thank you for the explanation, I did the update ✅ .